src/Uno.UI/UI/Xaml/Media/Animation/ExponentialEase.Android.cs

```diff
@@ -0,0 +1,12 @@
+using Android.Animation;
+
+namespace Windows.UI.Xaml.Media.Animation
+{
+	partial class ExponentialEase
+	{
+		internal override ITimeInterpolator CreateTimeInterpolator()
+		{
+			return EasingFunctionHelpers.GetPowerTimeInterpolator((float)Exponent, EasingMode);
```

Comment:
I get that the goal is just to avoid an exception, but this should probably be marked NotImplemented and/or log a warning, because a polynomial function is completely different from an exponential function.

Reply:
Having a wrong easing is better than an exception. If you know which would be the right one, you're welcome ;-)

Reply:
I think that I would just provide a `NullInterpolator` which directly does to the final value. I won't use another random interpolator.

Reply:
I'll replace it by a `LinearInterpolator`.